Kitodo/src/main/java/de/sub/goobi/forms/AktuelleSchritteForm.java

```diff
@@ -175,18 +178,14 @@ private String sortList() {
         if (this.sortierung.equals("prozessDesc")) {
             sort += ", " + SortBuilders.fieldSort("process").order(SortOrder.DESC).toString();
         }
-        /*if (this.sortierung.equals("batchAsc")) {
-            order = Order.asc("proc.batchID");
-        }
-        if (this.sortierung.equals("batchDesc")) {
-            order = Order.desc("proc.batchID");
-        }
-        if (this.sortierung.equals("prozessdateAsc")) {
-            order = Order.asc("proc.creationDate");
-        }
-        if (this.sortierung.equals("prozessdateDesc")) {
-            order = Order.desc("proc.creationDate");
-        }*/
+        /*
```

Comment:
Please remove this out commented code. It is the fifth or sixth time that this code get formatted.